Content.Client/Power/APC/UI/ApcMenu.xaml.cs

```diff
@@ -34,7 +37,7 @@ public void UpdateState(BoundUserInterfaceState state)
 
             if (PowerLabel != null)
             {
-                PowerLabel.Text = Loc.GetString("apc-menu-power-label", ("power", castState.Power));
+                PowerLabel.Text = castState.Power + "W";
```

Comment:
why remove the localization?

Reply:
> why remove the localization?

The old system was the power int was baked into to trans string. Now I removed it and the label trans is in the XAML and the .cs only manages the power. 

Question, I made a mistake and didn't add whitespace. What's the best way to add the whitespace between thevalue and string. I assume +" W" is frowned on?